packages/core/lib/datasources/Datasource.js

```diff
@@ -137,18 +137,25 @@ class Datasource extends EventEmitter {
     // Transform the received quads
     let destination = new BufferedIterator(), outputQuads, graph = this._graph;
     outputQuads = destination.map((quad) => {
+      let transformedSubject, transformedObject, transformedGraph;
```

Comment:
Alternative way of writing the method could be to

```suggestion
      let { subject, predicate, object, graph } = quad;
```

so you don't have to use `quad.` all the time below, and the `\\` from the `transformedQuad` can also be dropped.

Reply:
It is indeed not the most elegant solution, but a graph variable was already declared for that function, and I wanted to make sure not to introduce inconsistencies or make the code difficult to understand. 
I could update it, and rename the previously introduced graph variable (that one refers to the default graph that has been set if I recall correctly)

Reply:
I changed the custom default graph variable to _defaultGraph_ instead of _graph_, and used this approach in latest commit